README.md

```diff
@@ -49,6 +49,12 @@ With Standard Containers we can put an end to that embarrassment, by making INDU
 The specification and code is licensed under the Apache 2.0 license found in 
 the `LICENSE` file of this repository.  
 
+## Markdown style
+
+To keep consistency throughout the markdown files in the open container spec all files should be formatted one sentence per-line.
```

Comment:
Gruber [capitalizes “Markdown”][1].  And the OCI [capitalizes Open Container][2].

[1]: http://daringfireball.net/projects/markdown/
[2]: http://opencontainers.org/